modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/objectDefinitionUtil.ts

```diff
@@ -130,9 +130,21 @@ export async function deleteRelationship(id: number) {
 		});
 	}
 	catch (error) {
-		Liferay.Util.openToast({
-			message: (error as Error).message,
-			type: 'danger',
+		const errorMessage = (error as Error).message;
+		Liferay.Util.openModal({
+			bodyHTML: `<p>${errorMessage}</p>`,
+			buttons: [
+				{
+					displayType: 'warning',
+					label: Liferay.Language.get('done'),
+					type: 'cancel',
+				},
```

Comment:
@larissacribeiro why do we need these buttons? Are they performing any action?

Reply:
Hi @carolmariaabb! I followed the design pattern of the existing `warning modals` in the portal, which is what I've been asked to use. here follows an visual example:
![image_720](https://github.com/liferay-objects/liferay-portal/assets/82094673/159b3151-9863-4885-9475-af958097dab0)

Reply:
Thanks @larissacribeiro